content/en/docs/setup/production-environment/tools/kubeadm/install-kubeadm.md

```diff
@@ -220,7 +220,7 @@ sudo systemctl enable --now kubelet
   - You can leave SELinux enabled if you know how to configure it but it may require settings that are not supported by kubeadm.
 
 {{% /tab %}}
-{{% tab name="Fedora CoreOS" %}}
+{{% tab name="Fedora CoreOS/Flatcar Container Linux" %}}
```

Comment:
checked the other tabs and i guess we should separate the two distros with `, ` (comma + space) here.

Reply:
Agree, but comma or the word "or"? The others have the comma as separator when dealing with more than two distros, like: CentOS, RHEL or Fedora

Reply:
(will assume the "or" is better here, but can change later without any problem)

Reply:
"or" seems fine.